metaworld/policies/mujoco/sawyer_xyz/base.py

```diff
@@ -0,0 +1,67 @@
+import abc
```

Comment:
there's no need to nest the `policies` package so deep. you can just put all of these in `metaworld.policies`